test/metabase/query_processor_test/alternative_date_test.clj

```diff
@@ -225,15 +228,33 @@
                  (mt/rows (mt/dataset string-times
                             (qp/process-query
                               (assoc (mt/mbql-query times)
-                                     :middleware {:format-rows? false})))))))))
+                                     :middleware {:format-rows? false}))))))))
+
+      (testing "mongo only supports datetime"
+        (mt/test-drivers #{:mongo}
```

Comment:
I know the other tests above are doing it this way but we actually have `mt/test-driver` for just one specific driver.

```suggestion
        (mt/test-driver :mongo
```

Altho maybe having two different ways of doing it actually doesn't make sense... 🤔 

Either way that's the pattern we generally use for testing just one driver